src/bin/tikv-server.rs

```diff
@@ -97,9 +97,12 @@ fn build_raftkv(matches: &Matches,
     opts.set_max_write_buffer_number(5);
     opts.set_min_write_buffer_number_to_merge(2);
     opts.set_max_background_compactions(3);
-    opts.set_max_bytes_for_level_base(512 * 1024 * 1024);
+    opts.set_max_bytes_for_level_base(64 * 1024 * 1024);
```

Comment:
Why 64M? The level base should be approximate `write_buffer_size * min_write_buffer_number_to_merge * level0_file_num_compaction_trigger`, which is 512M.